specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ContainerApps.json

```diff
@@ -454,6 +454,11 @@
         "dapr": {
           "$ref": "#/definitions/Dapr",
           "description": "Dapr configuration for the Container App."
+        },
+        "maxInactiveRevisions": {
```

Comment:
Can we have default value for this?

Reply:
I think we should not have a default value here. Because we have a cluster level config, which is disabled for now. Displaying a default value can be misleading for users.